### PR TITLE
DF-1263 500 Error Page

### PR DESCRIPTION
**Pull request by sebworks**
Add 500 Error Page

## Changes
 
- Added ' _layouts/500.html' to add 500 page.
- Added ' _layouts/error.less' to add error page markup.
- Added ' _static/img/server-rack.png' to add 500 page assets.

## Review
 
- @jimmynotjim  
- @anselmbradford
 
## Testing

- Run sheer without debug mode : 'Sheer Serve'
-  Cause a 500 error by removing a '%' in blog/index.html
-  Navigate to 'http://localhost:7000/blog/'

## Notes
- Unsure on the breakpoints. There maybe a better way to handle how the image and action container display on tablet.

## Preview
 
screen shot 2015-04-01 at 10 45 52 am

screen shot 2015-04-02 at 4 55 10 pm

screen shot 2015-04-02 at 4 57 38 pm

### CODE REVIEW

File: _layouts/500.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+{%- endblock %}
+
+{% block content_modifiers -%}
+	error_main
+{%- endblock %}
+
+{% block content_main %}
+    <div class="error_copy_container">
+        <section class='error_heading u-mt30'>
```
Comment (jimmynotjim) [inline _layouts/500.html]: Should this be a `<header>` making the class `error_header`?

File: static/css/error.less [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    .respond-to-range(601px, 1199px, {
+        .grid_column(8);
+    });
+
+    .respond-to-max(600px, {
+        .grid_column(12);
+    });
+}
+
+.error_featured-img {
```
Comment (jimmynotjim) [inline static/css/error.less]: Is it possible to set the bg-image on the content wrapper rather than deal with absolute positioning images?

**Comment by jimmynotjim** [comment]
Do you have a screenshot of a small device view?

**Comment by sebworks** [comment]
@jimmynotjim, I'll add them for the tablet and mobile.

File: _layouts/500.html [hunk 3]
```diff
@@ -0,0 +1,48 @@
+{% extends "layout-full.html" %}
+
+{% block title -%}
+    500 internal server error
+{%- endblock %}
+
+{% block desc -%}
+
```
Comment (anselmbradford) [inline _layouts/500.html]: Just checking... this block is okay to be empty?

File: _layouts/500.html [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    <div class="error_copy_container">
+        <section class='error_heading u-mt30'>
+            <h1 class='superheader'>500 internal server error</h1>
+            <p class='short_desc'>We can’t find the page you were
+                looking for, but here are
+                a number of other ways you
+                might try to get there.
+            </p>
+        </section>
+        <aside class='block padded-container u-mt30 u-mb60 '>
```
Comment (anselmbradford) [inline _layouts/500.html]: Minor...
- Trailing whitespace.
- You may want to update this file to use double-quotes throughout per https://github.com/cfpb/front-end/blob/master/markup.md#quotes

File: _layouts/500.html (hunk 3 above)
Comment (sebworks) [inline _layouts/500.html]: I think it's empty on a number of pages but we should be populating it.

File: _layouts/500.html [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+            <div class="content-l content-l__large-gutters padded-container_body">
+                <div class="content-l_col content-l_col-1-2 ">
+                    <p>
+                        Explore our other useful tools and resources
```
Comment (anselmbradford) [inline _layouts/500.html]: Does this need a period at the end? The text in the column on the right has a period. This appears to be an error in the original mocks where the mobile sizes have a period and the desktop sizes don't.

Comment (sebworks) [inline _layouts/500.html]: @anselmbradford , I wish I had your glasses.

Comment (anselmbradford) [inline _layouts/500.html]: Haha, they're pretty thick :eyeglasses:. I'm ineligible for Lasik due to my prescription being too strong :sob:

File: static/css/error.less (hunk 2 above)
Comment (sebworks) [inline static/css/error.less]: @jimmynotjim , so I wasn't able to get it work with background-position due the breakpoints so I left it as an img tag.

**Comment by sebworks** [comment]
@anselmbradford , @jimmynotjim changes made and ready to be reviewed.

**Comment by sebworks** [comment]
@anselmbradford , yeah we probably could avoid that by resizing the image. I'll show it tomorrow.

**Comment by jimmynotjim** [comment]
Agreed, LGTM

**Pull request merged**